docs/roadmap.md

```diff
@@ -11,37 +11,49 @@ Here's how we intend to roll out Project Reunion:
 <table>
   <tbody>
     <tr>
-      <th>//BUILD 2020</th>
-      <th>End of 2020</th>
-      <th>2021++</th>
+      <th>Reunion 0.5 (March 2021)</th>
+      <th>Reunion 0.8 (Summer 2021)</th>
+      <th>Reunion 1.0 (Fall 2021)</th>
+      <th>2022 and Beyond</th>
     </tr>
     <tr>
-      <td>
+    <td>
+      <ul>
+            <li>1st supported public release</li>
+            <li>WinUI 3 to work in Desktop Packaged (Win32) Apps</li>
```

Comment:
Can you add a link from "Desktop Packaged (Win32)" to the docs.microsoft.com topic on Desktop Bridge packaging?